addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/ExtensionAscanRules.java

```diff
@@ -51,24 +48,5 @@ public boolean canUnload() {
     @Override
     public void hook(ExtensionHook hook) {
```

Comment:
This can be removed now (sorry).

Reply:
No need to be sorry, I shoulda seen that.